setup.py

```diff
@@ -6,7 +6,7 @@
     long_description = fh.read()
 
 setup(name='pipelinewise-tap-kafka',
-      version='7.1.0',
+      version='7.1.1',
```

Comment:
A new release needs a separate PR and updated Changelog.

Reply:
Since this is a tiny change, could I add the change-log to this PR instead of a new PR ?

Reply:
CI is failing because test coverage is decreased. it means this new implemented method needs at least a unit test to reach the test coverage limit. In that case it won't be a tiny PR :).
Do you want to merge it while Cl checks are failing?

Reply:
Ah thanks ... I notice that now.

This change is just a bug stop-gap until we implement partition selection ... partition selection will need much of the code to be re-written. I will increase test coverage for this by removing the new function

Reply:
We were discussing about release PR which should not be in the bugfix/feature PR and now I see the test coverage limit is decreased just to be passed!
Please never decrease the test coverage limit! 
I added a fake unit test just for passing the CI checks!